frontend/src/components/Toast.tsx

```diff
@@ -0,0 +1,42 @@
+import { useToast } from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+
+type ToastStatus = "error" | "warning" | "success" | "info" | undefined;
```

Comment:
It may be a better idea to use an enum here instead

Reply:
The type within chakra UI is explicitly `"error" | "warning" | "success" | "info" | undefined` creating an enum is causing type issues when calling the chakra UI toast

Reply:
Makes sense, thanks!